packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -168,6 +168,21 @@ abstract class ThemeExtension<T extends ThemeExtension<T>> {
 ///   * [Radio]
 ///   * [Switch]
 ///   * [Checkbox]
+
+/// Defines how tap targets are sized for Material widgets.
```

Comment:
What's this change for? This removes the the existing document (since they're separated from the class definition) and replaced with a new one.